flight/mcu/lib/teensyShared/rocketFSM.cpp

```diff
@@ -24,34 +24,28 @@
 #include <SPI.h>
 #include <Wire.h>
 
-#include "KX134-1211.h"       //High-G IMU Library
-#include "SparkFunLSM9DS1.h"  //Low-G IMU Library
 #include "acShared.h"
 #include "dataLog.h"
 #include "hybridShared.h"
 #include "pins.h"
 #include "thresholds.h"
 
-fsm_struct rocketTimers;
-
 rocketFSM::rocketFSM(pointers *ptr) { pointer_struct = ptr; }
 
 void rocketFSM::tickFSM() {
     // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
     // Lock mutexes for data used in switch
     chMtxLock(&pointer_struct->dataloggerTHDVarsPointer.dataMutex_lowG);
-    chMtxLock(&pointer_struct->dataloggerTHDVarsPointer.dataMutex_RS);
-    chMtxLock(&pointer_struct->dataloggerTHDVarsPointer
-                   .dataMutex_GPS);  // TODO, used for state init?
 
     // get the linear accelration from the lowgimu
     float linear_acceleration =
-        -pointer_struct->sensorDataPointer->lowG_data.ay;
+        pointer_struct->sensorDataPointer->highG_data.hg_az;
```

Comment:
why did we switch from low g to high g data here? Also we started using high g but are still mutex locking the low g mutex.

Reply:
highg tells us a broader range of g forces than lowg and we don't need extreme accuracy for the fsm code. Mutex thing needs to be fixed